build/configs/imxrt1050-evk/nxp_demo_protected/defconfig

```diff
@@ -875,6 +892,7 @@ CONFIG_EXAMPLES_NXP_DEMO=y
 # CONFIG_EXAMPLES_SPEECH_DETECTOR_TEST is not set
 # CONFIG_EXAMPLES_ST_THINGS is not set
 # CONFIG_EXAMPLES_TESTCASE is not set
+# CONFIG_EXAMPLES_UPDATE_TEST is not set
```

Comment:
Please enable the configs for MPU TEST

Reply:
Can we separate feature and test config, it can be used only during testing